### PR TITLE
tasks: downgrade-wpa-hostapd: Update urls

### PR DESCRIPTION
**Pull request by markfeathers**
Debian has stopped hosting these older debian packages.

### CODE REVIEW

File: tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh [hunk 1]
```diff
@@ -6,30 +6,30 @@ ROOT_DST="${DS_OVERLAY}/tmp/downgrade-wpa-hostapd"
 # Debian 12 (Bookworm)
 if [ "${DS_DISTRO}" == "debian" ] ; then
 	LIBSSL_DEB="libssl1.1_1.1.1w-0+deb11u1_armhf.deb"
-	LIBSSL_SITE="http://ftp.us.debian.org/debian/pool/main/o/openssl/${LIBSSL_DEB}"
+	LIBSSL_SITE="https://files.embeddedts.com/misc/${LIBSSL_DEB}"
 	LIBSSL_SHA256="42130c140f972d938d4b4a5ab9638675e6d1223fcff3042bbcc1829e3646eb00"
 
 	WPA_DEB="wpasupplicant_2.9.0-21_armhf.deb"
```
Comment (ts-kris) [inline tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh:12]:
```suggestion
	WPA_DEB="wpasupplicant_2.9.0-21+deb11u2_armhf.deb"
```

File: tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh [hunk 2]
```diff
@@ -6,30 +6,30 @@ ROOT_DST="${DS_OVERLAY}/tmp/downgrade-wpa-hostapd"
 # Debian 12 (Bookworm)
 if [ "${DS_DISTRO}" == "debian" ] ; then
 	LIBSSL_DEB="libssl1.1_1.1.1w-0+deb11u1_armhf.deb"
```
Comment (ts-kris) [inline tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh:8]: This URL is still valid.

**Review by randytsi369 — APPROVED** [review]
Looks good to me, not tested.

File: tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh [hunk 3]
```diff
@@ -6,30 +6,30 @@ ROOT_DST="${DS_OVERLAY}/tmp/downgrade-wpa-hostapd"
...
-	LIBSSL_SITE="http://ftp.us.debian.org/debian/pool/main/o/openssl/${LIBSSL_DEB}"
+	LIBSSL_SITE="https://files.embeddedts.com/misc/${LIBSSL_DEB}"
 	LIBSSL_SHA256="42130c140f972d938d4b4a5ab9638675e6d1223fcff3042bbcc1829e3646eb00"
 
 	WPA_DEB="wpasupplicant_2.9.0-21_armhf.deb"
-	WPA_SITE="http://ftp.us.debian.org/debian/pool/main/w/wpa/${WPA_DEB}"
+	WPA_SITE="https://files.embeddedts.com/misc/${WPA_DEB}"
 	WPA_SHA256="7bb05cb4f6e3fa3a12d10ea900a4bc7d7d606cacc254115ac28583903b55a2a0"
```
Comment (ts-kris) [inline tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh:14]:
```suggestion
	WPA_SHA256="dce8b75e6b4d6e8a59c98b2172d553de332026bc2f8705b63e12a1e980150a5b"
```

File: tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh [hunk 4]
```diff
@@ -6,30 +6,30 @@ ROOT_DST="${DS_OVERLAY}/tmp/downgrade-wpa-hostapd"
...
 
 	HOSTAPD_DEB="hostapd_2.9.0-21_armhf.deb"
```
Comment (ts-kris) [inline tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh:16]:
```suggestion
	HOSTAPD_DEB="hostapd_2.9.0-21+deb11u2_armhf.deb"
```

File: tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh [hunk 5]
```diff
@@ -6,30 +6,30 @@ ROOT_DST="${DS_OVERLAY}/tmp/downgrade-wpa-hostapd"
...
-	HOSTAPD_SITE="http://ftp.us.debian.org/debian/pool/main/w/wpa/${HOSTAPD_DEB}"
+	HOSTAPD_SITE="https://files.embeddedts.com/misc/${HOSTAPD_DEB}"
 	HOSTAPD_SHA256="e9bd2195d497e420dfda9a77726c5bc9f79315faee5633517912e06154fe59c2"
```
Comment (ts-kris) [inline tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh:18]:
```suggestion
	HOSTAPD_SHA256="67e82fc8753f7b6379bbeaad529170ec03ec3bba414c807a67512229a7cacdae"
```

File: tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh [hunk 6]
```diff
@@ -6,30 +6,30 @@ ROOT_DST="${DS_OVERLAY}/tmp/downgrade-wpa-hostapd"
...
 # Ubuntu distros shipping wpasupplicant 2.10 are:
 # 22.04 (Jammy)
 # 23.04 (Focal)
 elif [ "${DS_DISTRO}" == "ubuntu" ] ; then
 	LIBSSL_DEB="libssl1.1_1.1.1f-1ubuntu2_armhf.deb"
```
Comment (ts-kris) [inline tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh:23]: This URL is unchanged

File: tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh [hunk 7]
```diff
@@ -6,30 +6,30 @@ ROOT_DST="${DS_OVERLAY}/tmp/downgrade-wpa-hostapd"
...
-	LIBSSL_SITE="http://ports.ubuntu.com/pool/main/o/openssl/${LIBSSL_DEB}"
+	LIBSSL_SITE="https://files.embeddedts.com/misc/${LIBSSL_DEB}"
 	LIBSSL_SHA256="fde1628edbebc3b4aba18f2568b703a4c2003e4903c4e01f899b489f4e426d3f"
 
 	WPA_DEB="wpasupplicant_2.9-1ubuntu4.3_armhf.deb"
```
Comment (ts-kris) [inline tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh:27]:
```suggestion
	WPA_DEB="wpasupplicant_2.9-1ubuntu4.4_armhf.deb"
```

File: tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh [hunk 8]
```diff
@@ -6,30 +6,30 @@ ROOT_DST="${DS_OVERLAY}/tmp/downgrade-wpa-hostapd"
...
-	WPA_SITE="http://ports.ubuntu.com/pool/main/w/wpa/${WPA_DEB}"
+	WPA_SITE="https://files.embeddedts.com/misc/${WPA_DEB}"
 	WPA_SHA256="6d89d49615e903445f0321ddeec6e1d30cbc9fae6fe02cb6aa6f524ae0be969f"
```
Comment (ts-kris) [inline tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh:29]:
```suggestion
	WPA_SHA256="21108b6f429cf60644ee164c699f9dad60dbceb4f70e041583e8a390289f8db6"
```

File: tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh [hunk 9]
```diff
@@ -6,30 +6,30 @@ ROOT_DST="${DS_OVERLAY}/tmp/downgrade-wpa-hostapd"
...
 
 	HOSTAPD_DEB="hostapd_2.9-1ubuntu4_armhf.deb"
```
Comment (ts-kris) [inline tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh:31]: This URL is unchanged.

**Review by ts-kris — CHANGES_REQUESTED** [review]
I don't think we should take on this responsibility to re-host. We should instead grab the latest update URLs (not all changed) and go from there.

**Pull request merged**